projects/tibco-tcstk/tc-liveapps-lib/src/lib/services/tc-case-audit.service.ts

```diff
@@ -11,17 +11,32 @@ export class TcCaseAuditService {
 
   constructor(private http: HttpClient) { }
 
-  public getCaseAudit(caseRef: string, sandboxId: number, startAt: string, top: number): Observable<AuditEventList> {
+  public getCaseAudit(caseRef: string, sandboxId: number, startAt: string, top: number,
+    creationTime: string, caseType: CaseTypeAudit, orderBy: OrderBy): Observable<AuditEventList> {
```

Comment:
Suggest making the extra parameters optional so we don't break existing apps.   eg:) public getCaseAudit(caseRef: string, sandboxId: number, startAt: string, top: number,
    creationTime?: string, caseType?: CaseTypeAudit, orderBy?: OrderBy): Observable<AuditEventList> {

Reply:
Also only add to URL if they are passed...

Reply:
Feedback regarding optional values accepted and pushed in.

Isn't the code below sufficient at detecting if they values exist before adding it to the url? See lines 19, 21, and 24 below.